src/functions/process-stream-event.ts

```diff
@@ -5,6 +5,7 @@ import {
   Handler,
   SQSEvent,
   StreamRecord,
+// eslint-disable-next-line import/no-unresolved
```

Comment:
What's this for?

Reply:
Linter is complaining I don't have the package, even though it is importing from @types/aws-lambda package. It also doesn't let me change the import to be from the type package directly

![Screenshot 2024-05-29 at 12 07 40](https://github.com/dvsa/cvs-tsk-update-store/assets/6936472/0d02816a-07e6-42a1-99ce-4a22535e94f1)